translations/en-us.yaml

```diff
@@ -37,6 +37,13 @@ influencer_network:
     skip: Skip
     learn_more: Learn More
     dont_show: Don't show me this again
+processed_features:
+  slow: Rarely Responds
+  default: Responds within {length, plural, =1 {1 hour} other {# hours}}
+  lda_compliant: LDA Compliant
+  tooltip_slow: Does not always respond to emails received
+  tooltip_medium: Average influencer email response time
+  tooltip_fast: Average influencer email response time
```

Comment:
It's normal that they have the same description

Reply:
Yeah they do, I kept them separated because it was simpler to handle in the template

Reply:
Ok no problem :)